assets/company-admin/components/CompanyDetailsProductFilter.tsx

```diff
@@ -29,9 +29,10 @@ class CompanyDetailsProductFilterComponent extends React.PureComponent<any, any>
 
     getDropdownItems(filter: any) {
         const sectionIds = this.props.companySections[this.props.currentCompany._id].map((section: any) => section._id);
+        const currentCompanyProducts: Set<string> = new Set(this.props.currentCompany.products.map(({_id}: any) => _id));
```

Comment:
If `<component>.propTypes` and `React.PureComponent<any, any>` was replaced with ts interface, this would remove the need for `any` in new code (there's not many props for this one).

Reply:
Good point, forgot that we're converting everything